### PR TITLE
docs: Add more info to the update api files section

### PR DESCRIPTION
**Pull request by tcharding**
Done to try to further help new contributors and save maintainer time.

Add an example of using `just check-api` and an example `git` command for devs to create the api patch now required for all changes to the public API.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -200,6 +200,12 @@ All PRs that change the public API of `rust-bitcoin` must include a patch to
 the `api/` text files. This should be a separate, final patch to the PR
 that is the diff created by running `./contrib/check-for-api-changes.sh`.
 
+Or use `just`:
+```
```
Comment (storopoli) [inline CONTRIBUTING.md:204]:
```suggestion
```bash
```

add syntax highlighting?

**Comment by tcharding** [comment]
Added a new blank line before the '```rust' marker too like you taught me before.

**Comment by tcharding** [comment]
And read my diff before pushing ... and removed the indentation :)

**Review by apoelstra — APPROVED** [review]
ACK 8e67531cfea1a5bc79ae0bf121646a8d16a0a82b

**Review by apoelstra — APPROVED** [review]
ACK 0ae5be3e428628c0ac9c97e03648192b5d9b0c3c

**Pull request merged**